lighthouse-core/gather/driver.js

```diff
@@ -830,11 +879,20 @@ class Driver {
       };
     });
 
-    // Wait for load or timeout and run the cleanup function the winner returns.
+    // Wait for security issue, load or timeout and run the cleanup function the winner returns.
     const cleanupFn = await Promise.race([
```

Comment:
having this `cleanupFn` setup now is weird if they all can (and should) be cancelled regardless, but that can wait for the refactor too